test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -34,9 +34,7 @@ class StructuredFormDataBuilderSpec extends Spec {
       createForm(
         "field" -> "fieldValue"
       ),
-      objectStructure(
-        "field" -> textNode("fieldValue")
-      )
+      objectStructure(field("field", textNode("fieldValue"), Map.empty))
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.